selfdrive/car/gm/values.py

```diff
@@ -33,27 +33,26 @@ class CarControllerParams:
 
   def __init__(self, CP):
     # Gas/brake lookups
-    self.ZERO_GAS = 2048  # Coasting
```

Comment:
for future reference, this is the conversion:

`((2048 << 3) | (1 << 15) | (1 << 17)) * 0.125 - 22534`